.circleci/config.yml

```diff
@@ -55,8 +55,8 @@ jobs:
         conda activate base &&
         cd ${ILASTIK_ROOT}/ilastik-meta &&
         python ilastik/scripts/devenv.py create -n ${TEST_ENV_NAME}
-        -p pytest-qt ilastik-dependencies-no-solvers
-        -c ilastik-forge conda-forge defaults
+        -p pytest-qt ilastik-dependencies-no-solvers elf
+        -c ilastik-forge conda-forge cpape defaults
```

Comment:
We should decide how to handle the elf functionality. [elf](https://github.com/constantinpape/elf) is a small python only library I recently wrote to bundle functionality that I need in several downstream dependencies.
It's fairly small, so we could consider adding it to ilastik dependencies (and pushing it on ilastik-forge).
If you don't want to add another dependency, we could also copy the relevant code somewhere to ilastk.

Reply:
Yes, I want to bring it on conda forge as soon as we have nifty there.